components/Dropdown/Dropdown.jsx

```diff
@@ -212,14 +216,12 @@ const SelectedItemLabel = styled.span`
 `;
 
 const DropItemImage = styled.img`
-  float: right;
   width: ${DROPITEM_IMAGE_SIZE};
   height: ${DROPITEM_IMAGE_SIZE};
+  margin-left: ${spacing.xsmall}px;
 `;
 
-const DropItemLabel = styled.span`
-  vertical-align: middle;
```

Comment:
Another point:
I think the `vertical-align` is not needed anymore here since the `DropItem` has been defined with `align-items: center;`